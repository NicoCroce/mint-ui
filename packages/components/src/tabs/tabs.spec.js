import * as React from "react";
import { render } from "@testing-library/react";
import Tabs from "./index";

test("Tabs renders correctly", () => {
    const { TabPane } = Tabs;
     const handleChange = (key) => {
        console.log(key);
    }
    const { asFragment } = render(
         <Tabs defaultActiveKey="a" onChange={handleChange}>
            <TabPane tabKey="a" tab="Tab A">
                Content of Tab A
            </TabPane>
            <TabPane disabled={true} tabKey="b" tab="Tab B">
                Content of Tab B
            </TabPane>
            <TabPane tabKey="c" tab="Tab C">
                Content of Tab C
            </TabPane>
        </Tabs>
    );
    expect(asFragment()).toMatchSnapshot();
});