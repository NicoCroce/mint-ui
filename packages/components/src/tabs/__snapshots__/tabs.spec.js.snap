// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tabs renders correctly 1`] = `
<DocumentFragment>
  <div
    class="tabs__Wrapper-sc-1wwz877-0 jeLJYD"
  >
    <ul
      class="tabs__TabsList-sc-1wwz877-1 iziYNA"
      role="tablist"
      style="border-bottom: 1px solid #e8e8e8;"
      tabindex="0"
    >
      <li
        class="tabs__ItemTab-sc-1wwz877-2 gJOhrG"
      >
        Tab A
      </li>
      <li
        class="tabs__ItemTab-sc-1wwz877-2 dniHWo"
        disabled=""
      >
        Tab B
      </li>
      <li
        class="tabs__ItemTab-sc-1wwz877-2 jxtSMj"
      >
        Tab C
      </li>
    </ul>
    <div
      class="active-tab-content"
    >
      <div>
        Content of Tab A
      </div>
    </div>
  </div>
</DocumentFragment>
`;
